app/views/provisioning_templates/index.html.erb

```diff
@@ -27,7 +27,12 @@
                                          :permission => 'edit_provisioning_templates') %>
         </td>
         <td class="ellipsis"><%= combination provisioning_template %></td>
-        <td><%= provisioning_template.try(:template_kind) %></td>
+        <td>
+          <% tmpl = provisioning_template.try(:template_kind) %>
+            <% if tmpl %>
+              <%= link_to tmpl, provisioning_templates_path(:search => "kind = #{tmpl.name}"), :title => "#{TemplateKind.template_description[tmpl.name]}" %>
+            <% end %>
+        </td>
```

Comment:
The indent of line 32 is not needed actually. You can also do `if tmpl = xxx` if Rubocop won't complain.

Reply:
this is still valid concern, if you will combine if and assignment, make sure you wrap it in parentheses, like this
`<% if (tmpl == provisioning_template.try(:template_kind)) %>`